src/components/view/pages/payment/paymentPageView.ts

```diff
@@ -77,30 +79,42 @@ export class PaymentPageView extends View<CartDataType> {
 
     public afterRender(controller: Controller): void {
         super.afterRender(controller);
-
-        let validName: boolean = false;
-        let validAddress: boolean = false;
-        let validEmail: boolean = false;
-        let validTell: boolean = false;
-        let validCardName: boolean = false;
-        let validCardNumber: boolean = false;
-        let validCardDate: boolean = false;
-        let validCardCvv: boolean = false
-
+        
+        const valid: Validation = {
+            validName: false,
+            validAddress: false,
+            validEmail: false,
+            validTell: false,
+            validCardName: false,
+            validCardNumber: false,
+            validCardDate: false,
+            validCardCvv: false
+        };
+
+        function setInputValidity(input: HTMLElement | null, isValid: boolean) {
+            if (!input) return;
+        
+            if (isValid) {
+                input.classList.remove('invalid');
+                input.classList.add('valid');
+            } else {
+                input.classList.add('invalid');
+                input.classList.remove('valid');
+            }
+        }
```

Comment:
I would suggest to use 2 func instead: `makeInputValid` and `makeInputInvalid` or something like this, it will be more clear